<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="actionShift"/>
    <addaction name="actionRotation"/>
    <addaction name="actionScale"/>
    <addaction name="actionTrim"/>
    <addaction name="separator"/>
    <addaction name="actionZoom"/>
    <addaction name="actionMinimize"/>
    <addaction name="separator"/>
    <addaction name="actionFill"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionTechnical_Support"/>
   </widget>
   <widget class="QMenu" name="menuPaint">
    <property name="title">
     <string>Paint</string>
    </property>
    <addaction name="actionDraw_None"/>
    <addaction name="actionDraw_Line"/>
    <addaction name="actionDraw_Curve"/>
    <addaction name="actionDraw_Rectangle"/>
    <addaction name="actionDraw_Polygon"/>
    <addaction name="actionDraw_Circle"/>
    <addaction name="actionDraw_Oval"/>
    <addaction name="actionDraw_Cube"/>
    <addaction name="separator"/>
    <addaction name="actionColor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPaint"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDraw_None"/>
   <addaction name="actionDraw_Line"/>
   <addaction name="actionDraw_Curve"/>
   <addaction name="actionDraw_Rectangle"/>
   <addaction name="actionDraw_Polygon"/>
   <addaction name="actionDraw_Circle"/>
   <addaction name="actionDraw_Oval"/>
   <addaction name="actionDraw_Cube"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionShift"/>
   <addaction name="actionRotation"/>
   <addaction name="actionScale"/>
   <addaction name="actionTrim"/>
   <addaction name="separator"/>
   <addaction name="actionZoom"/>
   <addaction name="actionMinimize"/>
   <addaction name="separator"/>
   <addaction name="actionColor"/>
   <addaction name="actionFill"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/close.png</normaloff>:/images/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/about.png</normaloff>:/images/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTechnical_Support">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/support.png</normaloff>:/images/images/support.png</iconset>
   </property>
   <property name="text">
    <string>Technical Support</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/undo.png</normaloff>:/images/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/redo.png</normaloff>:/images/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionDraw_Line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/line.png</normaloff>:/images/images/line.png</iconset>
   </property>
   <property name="text">
    <string>Draw Line</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDraw_Curve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/curve.png</normaloff>:/images/images/curve.png</iconset>
   </property>
   <property name="text">
    <string>Draw Curve</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDraw_Polygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/polygon.png</normaloff>:/images/images/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Draw Polygon</string>
   </property>
  </action>
  <action name="actionDraw_Circle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/circle.png</normaloff>:/images/images/circle.png</iconset>
   </property>
   <property name="text">
    <string>Draw Circle</string>
   </property>
  </action>
  <action name="actionDraw_Oval">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/oval.png</normaloff>:/images/images/oval.png</iconset>
   </property>
   <property name="text">
    <string>Draw Oval</string>
   </property>
  </action>
  <action name="actionDraw_Cube">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/cube.png</normaloff>:/images/images/cube.png</iconset>
   </property>
   <property name="text">
    <string>Draw Cube</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/zoom.png</normaloff>:/images/images/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionTrim">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/trim.png</normaloff>:/images/images/trim.png</iconset>
   </property>
   <property name="text">
    <string>Trim</string>
   </property>
  </action>
  <action name="actionRotation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/rotation.png</normaloff>:/images/images/rotation.png</iconset>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
  </action>
  <action name="actionShift">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/shift.png</normaloff>:/images/images/shift.png</iconset>
   </property>
   <property name="text">
    <string>Shift</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/fill.png</normaloff>:/images/images/fill.png</iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/minimize.png</normaloff>:/images/images/minimize.png</iconset>
   </property>
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Color.png</normaloff>:/images/images/Color.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionDraw_None">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/none.png</normaloff>:/images/images/none.png</iconset>
   </property>
   <property name="text">
    <string>Draw None</string>
   </property>
  </action>
  <action name="actionDraw_Rectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/rectangle.png</normaloff>:/images/images/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Draw Rectangle</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/zoom_in_out.png</normaloff>:/images/images/zoom_in_out.png</iconset>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/select.png</normaloff>:/images/images/select.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/delete.png</normaloff>:/images/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
